source/github/github.go

```diff
@@ -29,14 +29,14 @@ var (
 )
 
 type Github struct {
-	client *github.Client
-	url    string
-
-	pathOwner  string
-	pathRepo   string
-	path       string
-	options    *github.RepositoryContentGetOptions
-	migrations *source.Migrations
+	Client *github.Client
```

Comment:
Use `WithInstance()` in `GithubEE.Open()` instead of exporting the fields.

Reply:
Oh. That's a good point 👍